.github/workflows/test-jdk-25.yml

```
@@ -32,6 +34,10 @@ jobs:
         run: |
           git config --global user.name "github-actions[bot]"
           git config --global user.email "github-actions[bot]@users.noreply.github.com"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)